src/reformat/coeffs.rs

```diff
@@ -41,26 +41,21 @@ pub fn calculate_yuv_coefficients(
 #[cfg(test)]
 mod tests {
     use super::*;
+    use crate::internal_utils::assert_f32_array;
```

Comment:
In this case it is not strictly necessary since the containing module has `use crate::*` and this module has `use super::*`.

If we didn't use the `crate::` prefix here, then it would be fragile since it could fail to build if either of those wildcard imports are changed in the future.

One of the clean ups i have in my list is to replace all the wildcard imports with specific imports that are being used. especially in files like this where we only use 2 or 3 types there is no real need to import `crate::*`.